js/noms/src/database.js

```diff
@@ -78,7 +65,24 @@ export default class Database {
     return true;
   }
 
-  async commit(datasetId: string, commit: Commit<any>): Promise<Database> {
+  // Commit updates the commit that ds points at. If parents is provided then the promise
```

Comment:
Can you make these JSDoc comments?

```js
/**
 * ...
 */
```

We should document all public methods.